static/app/test/specs/entities.spec.js

```diff
@@ -8,27 +8,102 @@ var Radio = require('backbone.radio');
 
 // App
 var EntitiesAPI = require('app/entities').API;
+var Move = require('app/entities').Move;
 var channel = Radio.channel('global');
 
 describe('Entities', function(){
     describe('API', function(){
         describe('initialization', function(){
-            it('registers an entities:spots request handler');
-            it('registers an entities:moves request handler');
-            it('registers an entities:move request handler');
-            it('registers an entities:move:reset request handler');
+            
+            it('registers an entities:spots request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
```

Comment:
Because you create this in every test, it would be better to consolidate it in a `beforeEach` statement. To make it available in every test, you can put it after the `describe('API', function(){...` statement.

```javascript
describe('API', function(){
    beforeEach(function(){
        this.entitiesAPI = new EntitiesAPI();
    });
```